01 - JavaScript Drum Kit/index-START.html

```diff
@@ -1,66 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>JS Drum Kit</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
is this even needed? is not everything UTF-8 these days?

Reply:
I guess UTF-8 is strongly suggested as charset encoding because it covers more than 10000 characters. The main advantage of UTF-8 is also that it is compatible with other languages. Since the introduction of HTML5, UTF-8 was sold as better charset for web developers to maximise compatibility to other languages. Before HTML5 other charsets were suggested as better choice (ISO, ASCII, Windows) but they all had limitations in terms of extent of characters covered and language compatibility.

Reply:
yep, and browsers give a warning if you do not specify. and I think older windows systems assume a different encoding.